wrappers/java/src/test/java/org/hyperledger/indy/sdk/utils/InitHelper.java

```diff
@@ -3,12 +3,10 @@
 
 import org.hyperledger.indy.sdk.LibIndy;
 
-import java.io.File;
-
 public class InitHelper {
 	public static void init() {
 
-		if (!LibIndy.isInitialized()) LibIndy.init(new File("./lib/libindy"));
+		if (!LibIndy.isInitialized()) LibIndy.init("./lib/");
```

Comment:
Personally, i prefer lazy loading without providing explicit LibIndy.init() API at all. If a user (or tester) needs to load the library from the specific place he can just modify LD path for the current process that can be easily done with environment variables on the most platforms.

What do you think about this?